spark-3.1-spanner-lib/src/test/resources/db/populate_ddl.sql

```diff
@@ -12,6 +12,7 @@ CREATE TABLE games (
   winner STRING(36),
   created TIMESTAMP,
   finished TIMESTAMP,
+  max_date DATE,
 ) PRIMARY KEY(gameUUID);
 
 CREATE TABLE players (
```

Comment:
This can be done in a follow up pr. 

Please create table by types, for example: 
CREATE TABLE IntTable(
  not_null_index INT64 NOT NULL,
  int_column INT64,
) PRIMARY KEY(not_null_index);

CREATE TABLE StringTable(
  not_null_index STRING<10> NOT NULL,
  max_string_col STRING<MAX>,
  max_string_col STRING<100>,
) PRIMARY KEY(not_null_index);

CREATE TABLE TimestampTable(
  not_null_index TIMESTAMP NOT NULL,
  timestamp_col TIMESTAMP,
) PRIMARY KEY(not_null_index);

Or creating a single table with all the types, for example: 
CREATE TABLE IntegrationTestTable(
  not_null_index INT64 NOT NULL,
  max_string_col STRING<MAX>,
  100_string_col STRING<100>,
   int_column INT64,
  timestamp_col TIMESTAMP,
) PRIMARY KEY(not_null_index);

CREATE TABLE StringIndexTable(
  string_index STRING<20> NOT NULL,
) PRIMARY KEY(string_index);